app/users/resources.py

```diff
@@ -0,0 +1,48 @@
+from flask import request
+from flask_restful import Resource
+import json
+
+from app.users.models import User
+from app import db
```

Comment:
Just to get used to our importing convention, we usually group our import into three stanzas. One for python modules, one for third party modules, and one for our own modules (and in alphabetical order). So it would look something like:
```
import json

from flask import request
from flask_restful import Resources

from app import db
from app.users.models import User
```
Here is our styling guide if you are curious: https://git.inkling.com/platform/inkling-style-guide/tree/master/py